src/server/graphql/queries/__tests__/activeProOrgCount.test.js

```diff
@@ -1,44 +0,0 @@
-import mockAuthToken from 'server/__tests__/setup/mockAuthToken';
-import MockDB from 'server/__tests__/setup/MockDB';
-import expectAsyncToThrow from 'server/__tests__/utils/expectAsyncToThrow';
-import activeProOrgCount from 'server/graphql/queries/activeProOrgCount';
-import {PRO} from 'universal/utils/constants';
-
-console.error = jest.fn();
-
-test('counts the number of Pro orgs', async () => {
-  // SETUP
-  const mockDB = new MockDB();
-  const {user} = await mockDB.init();
-  const authToken = mockAuthToken(user[1], {rol: 'su'});
-  // TEST
-  const initial = await activeProOrgCount.resolve(undefined, undefined, {authToken});
-
-  // VERIFY
-  expect(initial >= 0).toBe(true);
-});
-
-test('new Pro org increments counts of Pro orgs', async () => {
-  // SETUP
-  const mockDB = new MockDB();
-  const {user} = await mockDB.init();
-  const authToken = mockAuthToken(user[1], {rol: 'su'});
-  // TEST
-  await mockDB.newOrg({tier: PRO});
-  const next = await activeProOrgCount.resolve(undefined, undefined, {authToken});
-
-  // VERIFY
-  expect(next >= 1).toBe(true);
```

Comment:
this was the cheapest way i figured i could rewrite these tests to support parallel testing

Reply:
Ok, I made one small change to check for changes in the count instead (`!==`). Not ideal, but better. We'll move to another strategy in the future if we need to.